recipe/meta.yaml

```diff
@@ -34,18 +36,29 @@ requirements:
     - toolchain
     - boost-cpp 1.63.*
     - cmake
-    - zlib 1.2.*
-    - snappy
-    - brotli
-    - thrift-cpp
+    - zlib 1.2.*        # [unix]
+    - snappy            # [unix]
+    - brotli            # [unix]
+    - thrift-cpp        # [unix]
+    - python            # [win]
```

Comment:
Cool, thanks. It would be great if in this recipe we could set `%PARQUET_BUILD_TOOLCHAIN%` and that would be enough